modules/core/src/main/java/org/apache/ignite/startup/cmdline/CommandLineStartup.java

```diff
@@ -62,10 +82,50 @@
     /** Quite log flag. */
     private static final boolean QUITE;
 
+    /** Command to print Ignite system properties info. */
+    static final String PRINT_PROPS_COMMAND = "-systemProps";
+
+    /** Classes with Ignite system properties. */
+    static final List<Class<?>> PROPS_CLS = new ArrayList<>(Arrays.asList(
+        IgniteSystemProperties.class,
+        ExchangeContext.class,
+        GridCacheMapEntry.class,
+        LocalDeploymentSpi.class,
+        GridCacheDatabaseSharedManager.class,
+
+        // TODO: GG-34386
```

Comment:
this TODO refers to itself which does not seem correct.

Reply:
Removed